File: BurgerbarOrderWebAppAngular/BurgerbarOrderWebApp/src/app/models/contactData.ts
export class ContactData{
    id: number=1;
    phoneNumber: String;
    email: String;
    address: String;
    description: String;

    constructor(phoneNumber: string, email: string, address: string, description: string){
        this.phoneNumber=phoneNumber;
        this.email=email;
        this.address=address;
        this.description=description;
    }
}

File: BurgerbarOrderWebAppSpringBoot/src/main/java/danielklarenbach/burgerbarorderwebapp/Models/ContactData.java
package danielklarenbach.burgerbarorderwebapp.Models;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@Entity
@NoArgsConstructor
public class ContactData {
    @Id
    private int id;
    private String phoneNumber;
    private String email;
    private String address;
    @Lob
    @Column(length = 1000000)
    private String description;
}
